api/handlers.go

```diff
@@ -281,11 +281,12 @@ func (si *ServerImplementation) LookupAccountAppLocalStates(ctx echo.Context, ac
 		return notFound(ctx, errValueExceedingInt64)
 	}
 
+	requestedLimit := uintOrDefaultValue(params.Limit, si.opts.DefaultApplicationsLimit)
```

Comment:
I'd prefer the was `effectiveLimit` or just `limit`, since the default thing means it may not be the "requested" limit.